src/main/java/com/pet/common/config/SecurityConfig.java

```diff
@@ -74,19 +79,31 @@ public OAuth2AuthenticationSuccessHandler oauth2AuthenticationSuccessHandler(
         return new OAuth2AuthenticationSuccessHandler(jwt, accountService);
     }
 
+    @Bean
+    public AuthenticationEntryPoint authenticationEntryPoint() {
+        return new CustomAuthenticationEntryPoint();
+    }
+
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
             .authorizeRequests()
-            .antMatchers("/api/v1/shelter-posts").hasAnyRole("USER", "ANONYMOUS")
+
+            // 보호소 게시글
+            .antMatchers(v1("/shelter-posts")).hasAnyRole(ROLE_USER, ROLE_ANONYMOUS)
```

Comment:
오우 이부분도 해주셨군요!! 
get을 제외한 것들은 ROLE_USER여야 할 것 같은데
혹시 요러케 메소드까지 수정 요기 pr에서 부탁드려도 될까요?? ㅎㅎ.. 🥲 

```suggestion
            .antMatchers(HttpMethod.GET, v1("/shelter-posts")).hasAnyRole(ROLE_USER, ROLE_ANONYMOUS)
            .antMatchers(HttpMethod.POST, v1("/shelter-posts")).hasRole(ROLE_USER),
            .antMatchers(HttpMethod.DELETE, v1("/shelter-posts")).hasRole(ROLE_USER)
```